### PR TITLE
Display comments in view contribution page.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: marketpulse/main/templates/fxosprice_view.html [hunk 1]
```diff
@@ -68,6 +68,12 @@ <h3 class="title">Contribution</h3>
         {% endfor %}
       </div>
     </div>
+    <div class="col-md-6">
+      <div class="contribution-label">
+        <legend>Comments</legend>
+        <div> {{ contribution.comment }} </div>
+      </div>
+    </div>
```
Comment (comzeradd) [inline marketpulse/main/templates/fxosprice_view.html]: You don't need to create another div with the `col-md-6` div. Just include comments inside the one above.

**Comment by akatsoulas** [comment]
@comzeradd pr updated.

File: marketpulse/main/templates/fxosprice_view.html [hunk 2]
```diff
@@ -66,6 +66,8 @@ <h3 class="title">Contribution</h3>
           <div>Price: {{ plan.amount }} {{ plan.currency }}</div>
           <hr>
         {% endfor %}
+        <legend>Comments</legend>
+        <div> {{ contribution.comment }} </div>
```
Comment (comzeradd) [inline marketpulse/main/templates/fxosprice_view.html]: You could nest this to its one `contribution-label` div.

**Comment by comzeradd** [comment]
r+wc :rocket:

**Pull request merged**